app/views/govuk_publishing_components/components/docs/step_by_step_nav_related.yml

```diff
@@ -62,3 +62,17 @@ examples:
           tracking_id: 'this-is-the-tracking-id'
         }
       ]
+  always_display_as_a_list:
+    description: |
+      Related links part of a step by step where list only includes one always show as a header. Sometimes this is not desirable. 
```

Comment:
I think this sentence is a bit unclear, perhaps consider re-wording it to something like: `when this component is rendered alongside an expanded step by step, we want it to render as a heading. However, in some cases the component will not be followed by a step by step. [maybe add your example here of the 2 step-by-steps]. In these cases, the heading is not followed by any content, so it should be rendered as a list instead.`